static/app/components/eventsTable/eventsTableRow.spec.jsx

```diff
@@ -18,4 +35,49 @@ describe('EventsTableRow', function () {
     );
     expect(container).toSnapshot();
   });
+
+  it('renders the replay column with a correct link', () => {
+    const event = TestStubs.DetailedEvents()[0];
+
+    render(
+      <table>
+        <tbody>
+          <EventsTableRow
+            organization={TestStubs.Organization()}
+            tagList={[
+              {
+                key: 'replayId',
+                name: 'Replayid',
+                totalValues: 5,
+              },
+            ]}
+            {...{orgId: 'orgId', projectId: 'projectId', groupId: 'groupId'}}
+            event={{
+              ...event,
+              tags: [
+                {
+                  key: 'replayId',
+                  value: 'test-replay-id',
+                },
+              ],
+            }}
+          />
+        </tbody>
+      </table>
+    );
+
+    expect(Button).toHaveBeenCalledWith(
```

Comment:
If we're using `render()` we shouldn't have to mock the button and inspect props, we should be looking at the actual html that was rendered.

Some combo of `getElementByRole` and `toHaveAttribute` should do it

Reply:
I won't be able to test the link value if I do it that way because in the test environment it gets an empty string as the href value when I pass an object to the `to` prop of link

Reply:
We can make it work by calling `initializeOrg` and passing in some context to the `render()` method. 
Last week we would've also needed to wrap things with `<RouteContext.Provider>` but now that context can be passed in as well.

Checkout https://gist.github.com/ryan953/7a554d9b369cd90d4d012d5c1a6207f1